ext/pcntl/pcntl.stub.php

```diff
@@ -25,6 +25,49 @@
  */
 const WCONTINUED = UNKNOWN;
 #endif
+#ifdef WEXITED
+/**
+ * @var int
+ * @cvalue LONG_CONST(WEXITED)
+ */
+const WEXITED = UNKNOWN;
+#endif
+#ifdef WSTOPPED
+/**
+ * @var int
+ * @cvalue LONG_CONST(WSTOPPED)
+ */
+const WSTOPPED = UNKNOWN;
+#endif
+#ifdef WNOWAIT
+/**
+ * @var int
+ * @cvalue LONG_CONST(WNOWAIT)
+ */
+const WNOWAIT = UNKNOWN;
+#endif
+
+/* First argument to waitid */
+/**
+ * @var int
+ * @cvalue LONG_CONST(P_ALL)
+ */
+const P_ALL = UNKNOWN;
+/**
+ * @var int
+ * @cvalue LONG_CONST(P_PID)
+ */
+const P_PID = UNKNOWN;
+/**
+ * @var int
+ * @cvalue LONG_CONST(P_PGID)
+ */
+const P_PGID = UNKNOWN;
+/**
+ * @var int
+ * @cvalue LONG_CONST(P_PIDFD)
+ */
+const P_PIDFD = UNKNOWN;
```

Comment:
that s a linux extension.

Reply:
Good catch, thanks. I added a #ifdef __linux__ as guard there.

Reply:
nope. ifdef P_PIDFD is better. see the man page. old kernels do not support it and we deal with centos 7/8 users.

Reply:
Good to know. I did originally try checking for the constant itself, but for some reason it doesn't work for me, preprocessor seems to not see the constant and skips the whole ifdef block block:

```
 #ifdef P_PIDFD
/**
 * @var int
 * @cvalue LONG_CONST(P_PIDFD)
 */
const P_PIDFD = UNKNOWN;
#endif
```

Reply:
I see, maybe it s because of [this inclusion](https://github.com/php/php-src/blob/e23bc2d50c3663d0c04a02902223ba48b9932df5/ext/pcntl/pcntl.c#L38). But regardless I ll test your branch locally later tonight. cheers.

Reply:
Quick smoke test:

```
php_error_docref(NULL, E_WARNING, "P_PIDFD is defined as: %d", P_PIDFD);                                                
#ifndef P_PIDFD                                                                                                         
php_error_docref(NULL, E_WARNING, "P_PIDFD is not defined");                                                            
#endif  
```

yields

```
Warning: pcntl_waitid(): P_PIDFD is defined as: 3 in /home/random/code/php/waitid/waitid.php on line 32

Warning: pcntl_waitid(): P_PIDFD is not defined in /home/random/code/php/waitid/waitid.php on line 32
```

Reply:
Looks like these constants are defined in `<linux/wait.h>` so would have to pull that header in first (but only if on Linux) in order to check them.